### ISSUE TITLE
[Settings - Startup]: JAWS is not announcing state as 'Selected' for Selected 'Startup', 'Interaction' etc button.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#13048]
### Windows Terminal version

1.13.10984.0

### Windows build number

1.13.10984.0

### Other Software

**Test Environment:**
**OS:** Windows 11 Version 21H2 (OS Build 22598.200)
**Screen Reader:** JAWS

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[JAWS is not announcing state as 'Selected' for Selected 'Startup', 'Interaction' etc button..zip](https://github.com/microsoft/terminal/files/8639090/JAWS.is.not.announcing.state.as.Selected.for.Selected.Startup.Interaction.etc.button.zip)

### Steps to reproduce

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Select 'Startup', 'Interaction' etc button.
Verify that JAWS is not announcing state as 'Selected' present under setting page.

### Expected Behavior

JAWS should announce state as 'Selected' for Selected 'Startup', 'Interaction' etc button.

### Actual Behavior

JAWS is not announcing state as 'Selected' for Selected 'Startup', 'Interaction' etc button.
